src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -2027,7 +2031,7 @@ End Module").Path
             ' - update project-system to recognize the new value and pass it through
```

Comment:
The line about IDE drop-down can be removed (the drop-down was removed for C#, so won't be added for VB) and I think we didn't need to modify project-system repo anymore (I don't see a PR there for 15.3 or 15.6).